tests/integration/framework/process/kubernetes/store/store.go

```diff
@@ -45,6 +45,17 @@ func (s *Store) Add(objs ...client.Object) {
 	}
 }
 
+func (s *Store) Delete(objs ...client.Object) {
+	s.lock.Lock()
+	defer s.lock.Unlock()
+	if s.objs == nil {
+		s.objs = make(map[string]client.Object)
```

Comment:
Indeed, though semantically it's no different and I think the optimization to machine code (if there is any) is academic, but have added :+1: 

The only reason I didn't add an early return here is it's 1 line of code shorter without :shrug:

Reply:
Yeah, it's not a big deal, since objs is mostly going to be small in our use case, but we can save some loop cycles in case we ever send bigger objs as an argument.

Reply:
I actually would have hoped the Go compiler/runtime to produce the same result, but who knows :slightly_smiling_face:

Reply:
Oh, I had no idea it would do that. TIL! 🙏  :)